test/livebook_web/live/session_live_test.exs

```diff
@@ -412,6 +412,19 @@ defmodule LivebookWeb.SessionLiveTest do
   describe "runtime settings" do
     test "connecting to elixir standalone updates connect button to reconnect",
          %{conn: conn, session: session} do
+      runtime_modules = Application.get_env(:livebook, :runtime_modules)
+
+      Application.put_env(:livebook, :runtime_modules, [
+        Livebook.Runtime.ElixirStandalone,
+        Livebook.Runtime.MixStandalone,
+        Livebook.Runtime.Attached,
+        Livebook.Runtime.Embedded
+      ])
```

Comment:
Since we use `Embedded` a lost in tests, let's move this configuration to `test_helpers.exs`, so that it applies everywhere :)

Sidenote: changing app config in a specific test is prone to race conditions if other tests would do that and use `async: true`.